src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -89,7 +89,7 @@ public void onBuildTriggered(AbstractProject<?, ?> project, String commitSha, bo
         String triggeredStatus = getDescriptor().getTriggeredStatusDefault(this);
 
         // check if we even need to update
-        if (triggeredStatus.equals("--none--")) {
+        if (triggeredStatus != null && triggeredStatus.equals("--none--")) {
```

Comment:
Ok, so let's say there is a default and it's always "--none--". Then why null-check? If default can be `null`, then you will continue running with triggeredStatus being null, which I expect will result in NPE further down the road.

Is it your intention?